seisidd/config/scan_template.yml

```diff
@@ -0,0 +1,103 @@
+# NOTE:
+# This is a yaml based scan configuration file that can be directly pass
+# to the tomo_scan() function or nf_scan(), ff_scan()
+# i.e.
+# RE(tomo_scan(SCAN_CONFIG.YML))
+
+# Will use a centralized yml for all the scans, notebook will spit out individual scan configs/pdf
+
+
+
+tomo:
+  ## Beamline optics
+  energyfoil:   "Re"      #   Ho, Pb, Re, Tb, Air, Au, Yb, Bi, Pt, Ir, Hf, Tm, Ta
+                          #   probably just need to do once???
+  attenuation:  "a number"         # Place hodler. This doesn't seem to be pre-defined for the entire run
+  FS_control:   "sweep"   # sweep, detpulse, detexposure etc. used this to determine FS control
+  beamsize_h:   0.5       # horizontal beam size
+  beamsize_v:   0.5       # vertical beam size
+  ## Scan parameters
+  type:    fly           # [step|fly]
+  n_white: 5             # num of white field before and after tomo
```

Comment:
This is a good point. I think we need to minimize the user input to avoid confusion (and typos) here and there. 
I will keep them there for now but make a comment so that we remove all the non-important/constant inputs later.